### ISSUE TITLE
Inconsistency of the results for monopole 1D tutorial

### ISSUE THREAD

**Issue by wyj216** [unicfdlab/libAcoustics#32]
Dear developers,
Thank you very much for this library. I have just compiled it and run the simplest tutorial. However, there are obvious discrepancies between the pressure oscillation amplitudes obtained with FWH and the ones probed in the computational domain. Therefore I have also checked the implemented formulas, i.e. the function Foam::functionObjects::Farassat1AFormulation::observerAcousticPressure(label iObs). The formulas do not seem to be correct.
1. in many places OneByOneMrSq is multiplied, but it should be divided;
2. dotUn in the code is already dotU & n + U & dotn, the meaning is not the same as in the popular formula. The same for dotMr, dotlr.

However, in the current tutorial, dotn, dotr should be zero, OneByOneMrSq should be one. So these things do not influence the current result. I thought it may due to the wedge boundary condition. 
For FWH, the Sphere boundary should be a complete sphere surface. How could the effect of the neglected part of the sphere be considered in the code? I could not figure it out by myself.

**Comment by unicfdlab** [comment]
Hello, thank you for the feedback. We have reviewed our code several time and I don't think it contains severe errors, but I'll be happy to verify it again with you.
Meanwhile, FWH must be applied to the 3D sphere (without any symmetry assumptions) if you want to validate the implementation using the monopole source. Can you post here the description of your test case?

**Comment by wyj216** [comment]
Hello, thank you for the quick response. I hope, it is my fault. I used the branch for OF-V1912. The mentioned case is a case in the tutorial, i.e. monopole 1D. In this tutorial, without changing anything, Farassat1AFormulation is used, and there is also symmetry assumption in this case. That might be the reason why there is inconsistency between postProcessing/probes/0/p and acousticData/monopoleFwhTest-time.dat, I mean the amplitude, not the absolute value.

Regarding the formula, I will try to find out some references and discuss it here later.

**Comment by wyj216** [comment]
Hello, there are too many sources for the formula, and the symbol meaning could be different. Here I take a summary document of F. Farassat as the reference, i.e. "Derivation of Formulations 1 and 1A of Farassat". https://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/20070010579.pdf
However, we can use any reference as you prefer. In this document, for example, equations (67) and (68) are the final formulas. For the calculation of p'_T (qds_ in the code), in the second part, OneByOneMrSq * OneByOneMr should be divided, but in the code, it is multiplied (line 204 in Farassat1AFormulation.C). The same for fpart1, fpart2, fpart3.
Also, in this document, dot(vn) is already, dot v*n + v * dotn, als dotUn in the code, then there is no more v * dotn. In the code, there is still (U & dotn), it seems that it has been included twice.
I hope it is my misunderstanding somewhere. I just want to help if possible, and not mean to critisize. I really appreciate your effort to develop such a liborary.

**Comment by unicfdlab** [comment]
> OneByOneMrSq * OneByOneMr should be divided, but in the code, it is multiplied

OneByOneMr is 1/(1 + ....). That's why I multiply by this term

**Comment by wyj216** [comment]
> > OneByOneMrSq * OneByOneMr should be divided, but in the code, it is multiplied
> 
> OneByOneMr is 1/(1 + ....). That's why I multiply by this term

Exactly! sorry, that's really my misunderstanding.

**Comment by unicfdlab** [comment]
> dotUn in the code is already dotU & n + U & dotn, the meaning is not the same as in the popular formula. The same for dotMr, dotlr.

Regarding this issue you are right - I will correct the source code and will come back in few minutes

**Comment by unicfdlab** [comment]
Yes, you are correct, I have updated the source code (see also attached paper with designations, similar to the source code - https://arc.aiaa.org/doi/abs/10.2514/6.2010-3711). Nevertheless, these corrections don't make sense to your test case, because they work only for moving observer or moving control surface

**Comment by unicfdlab** [comment]
Thank you for reviewing the code

**Comment by wyj216** [comment]
Yes, you are correct. These corrections do not make sense to my test case. Now I am running monopole 3D case. I will check the result with the real 3D calculation.
